presto-base-jdbc/pom.xml

```diff
@@ -18,6 +18,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-main</artifactId>
```

Comment:
`JdbcRecordSink` receives a packed date with a 12-bit time zone key and it needs to look up the `DateTimeZone` for that key. What's the proper way to do that without consulting `DateTimeZoneIndex`?